src/extra/AutoUI/index.tsx

```diff
@@ -268,10 +268,12 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 
 	return (
 		<Flex
+			height="100%"
```

Comment:
Swapped the height for a flex
Keeping this conversation open for the discussion on accepting and destructuring the FlexProps as well as for the PS you left at the bottom

Reply:
The reasoning against a hardcoded `flex={1}` is that it would be unexpected in case this was used inside a Flex parent with more children. Eg:
```
<Flex height="100vh">
    <Box flex={1}>Huge graphic here, maybe a big map</Box>
    <AutoUI /> {/* trying to render a short table below the big graphic */}
</Flex>
```
In this case the two components would of course expand to the same height, but it will not be clear why.